.idea/ant.xml

```diff
@@ -1,14 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
+<project version="5">
   <component name="AntConfiguration">
-    <buildFile url="file://$PROJECT_DIR$/help/en/build.xml">
-      <customJdkName value="1.8" />
-    </buildFile>
-    <buildFile url="file://$PROJECT_DIR$/help/fr/build.xml">
-      <customJdkName value="1.8" />
-    </buildFile>
-    <buildFile url="file://$PROJECT_DIR$/build.xml">
-      <customJdkName value="1.8" />
-    </buildFile>
+    <buildFile url="file://$PROJECT_DIR$/help/en/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/help/fr/build.xml" />
+    <buildFile url="file://$PROJECT_DIR$/build.xml" />
```

Comment:
These lines no more define a custom JDK name. Is that desired? I'm just checking since I don't use Idea and doesn't know if this matters.